keyboards/spaceman/2_milk/keymaps/copypasta_macfancy/config.h

```diff
@@ -0,0 +1,4 @@
+// Copyright 2022 Ryan Neff (@JellyTitan)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#define TAPPING_TERM 200
```

Comment:
```suggestion
#pragma once

#define TAPPING_TERM 200
```

Reply:
Added pragma and moved tapping term below